recipes/norbert/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "norbert" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/sigsep/norbert.git
+    git_tag: "v{{ version }}"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```

I think this can be noarch? See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds
```suggestion
  noarch: python
```

---

As for `musdb` package, the official support for this library is 3.6 and 3.7.

---

When using noarch (and only when using noarch!) you constrain the Python version directly instead of using `skip`, see my other comments.

---

Oh ok i understand better the other review about Python constraint.

---

Done, resolving the conversation